README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
```

Comment:
Add usage examples or a brief guide on how to use 'task install' to ensure users understand what dependencies it will install.